packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -5767,31 +5816,31 @@ const tests = {
         {
           message:
             "The 'handleNext' function makes the dependencies of useEffect Hook " +
-            '(at line 13) change on every render. To fix this, wrap the ' +
-            "'handleNext' definition into its own useCallback() Hook.",
+            '(at line 13) change on every render. To fix this, wrap the construction of ' +
+            "'handleNext' in its own useCallback() Hook.",
           // Normally we'd suggest moving handleNext inside an
           // effect. But it's used more than once.
           // TODO: our autofix here isn't quite sufficient because
           // it only wraps the first definition. But seems ok.
           suggestions: [
             {
               desc:
-                "Wrap the 'handleNext' definition into its own useCallback() Hook.",
+                "Wrap the construction of 'handleNext' in its own useCallback() Hook.",
               output: normalizeIndent`
-                function MyComponent(props) {
-                  let handleNext = useCallback(() => {
-                    console.log('hello');
-                  });
-                  if (props.foo) {
-                    handleNext = () => {
-                      console.log('hello');
-                    };
-                  }
-                  useEffect(() => {
-                    return Store.subscribe(handleNext);
-                  }, [handleNext]);
-                }
-              `,
+          function MyComponent(props) {
```

Comment:
Indentation?

Reply:
Fixed in https://github.com/facebook/react/pull/19590/commits/eaa2aff9424e7ded164799fa2aef27ccfa4ed3d0